LoopbackApp/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.4.1)
+project(loopback LANGUAGES C CXX)
+
+# set up common compile options
+add_library(${PROJECT_NAME} SHARED
+    sles.cpp
```

Comment:
If we used a different pathname here, then the rename would not be necessary right?
I would rather not rename unless we absolutely have to, as it makes the git history a little harder to follow.

Reply:
this is kind of recommended: in the android build history, before experimental, there is a build version that take source for "jni" folder and convert them into Android.mk thing, then build; then comes experimental plugin; then comes CMake, and Android.mk  It is long history.  We like developers to move away from jni folder for that reason -- in some version of Android Studio, it takes "jni" folder and use old build system, creating troubles.